src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.ts

```diff
@@ -154,4 +154,10 @@ export class BootstrapCreateModalComponent implements OnDestroy, OnInit {
     });
     taskObs.subscribe(undefined, finishHandler, finishHandler);
   }
+
+  copyToClipBoard(token: any) {
```

Comment:
Could you please add an unit tests for this method?

Reply:
method removed